services/frontend-service/src/assets/_components.scss

```diff
@@ -41,5 +41,6 @@ Copyright 2023 freiheit.com*/
 @import 'src/ui/components/Spinner/Spinner';
 @import 'src/ui/components/EnvironmentLockDisplay/EnvironmentLockDisplay';
 @import 'src/ui/components/ReleaseVersion/ReleaseVersion';
+@import 'src/ui/components/RolloutStatusDescription/RolloutStatusDescription.scss';
```

Comment:
is there a reason why we need to add ".scss" here?
If not, please remove the ending, so that the imports look uniform.
If yes, please add a comment why this is necessary.

Reply:
No, there's no reason, just an oversight. Removed it.